source/include/mqtt_agent_command_functions.h

```diff
@@ -127,7 +127,7 @@ MQTTStatus_t MQTTAgentCommand_Publish( MQTTAgentContext_t * pMqttAgentContext,
  * @brief Function to execute for a SUBSCRIBE command.
  *
  * @param[in] pMqttAgentContext MQTT Agent context information.
- * @param[in] pSubscribeArgs Arguments for MQTT_Subscribe().
+ * @param[in] pVoidSubscribeArgs Arguments for MQTT_Subscribe().
```

Comment:
Should these (throughout the file) reference the doxygen tag for the coreMQTT API function?
```suggestion
 * @param[in] pVoidSubscribeArgs Arguments for @ref MQTT_Subscribe().
```

Reply:
Also, why the presence of `Void` in the parameter name?

Reply:
No, since we haven't figured out yet if we want to keep linking to coreMQTT or remove those references.

The reason for void is to easily distinguish between variables when we cast them to the correct pointer type, since these parameters are passed as `void *`. I updated `MQTTAgentCommand_Connect` to also use this naming, but not `MQTTAgentCommand_Publish` since that one doesn't use a custom struct that needs to be dereferenced here, it can just be cast when passing to MQTT_Publish()

Reply:
Why does that distinguishing matter from an API perspective? Isn't that an implementation detail?

Reply:
(This function isn't even part of the API, I don't think variable naming matters too much)

Reply:
I don't see how much value is `Void` naming pattern is helping for the implementation to be conveyed about the parameter handling. If the information has to be conveyed to the command function implementation, I think it should be conveyed in a stronger way (either at least through doxygen documentation Or through a separate parameter mentioning the type). 
However, I prefer that this information be encapsulated within the implementation than be shown in the header file.